### ISSUE TITLE
Remove rxjs-compat and properly update to a newer rxjs version

### ISSUE THREAD

**Issue by DanielHabenicht** [zxing-js/ngx-scanner#118]
**Describe the bug**
rxjs/compat is a package meant to provide an easy way to upgrade to a newer rxjs version. It provides backwards compatibility in order to run rxjs 5 and 6 code in parallel. 
Including rxjs in your bundle greatly effects the bundle size [and is not recommended](https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/v6/migration.md#backwards-compatibility).

**Expected behavior**
Remove rxjs and update the code to a newer rxjs version. (Mostly changing imports)

This effects all apps using ngx-scanner and is a pain in the ass. Just imagine every package doing something like this. 
Maybe I will provide a PR.

**Comment by odahcam** [comment]
It's already done, but we didn't publish it yet, there's a few things we gotta do first.

I'm closing it as a dupe of #117.

Thanks for the report and please subscribe to #119 (or v1.2.0 milestone) for more info. 😉

**Comment by DanielHabenicht** [comment]
Ohh, ok. My bad. I've just seen this [PR](https://github.com/zxing-js/ngx-scanner/pull/86) recently. As I viewed it again now I also see that it never went through.
Sry!

**Comment by odahcam** [comment]
No problem, thanks for contributing!
